### ISSUE TITLE
Spyder error

### ISSUE THREAD

**Issue by janvanlooy** [spyder-ide/spyder#6327]
## Description

**What steps will reproduce the problem?**

1. trying to finish statement grouped.gamescore.plot... with two brackets and error pops up
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\base_frontend_mixin.py", line 163, in _dispatch
    handler(msg)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\namespacebrowser.py", line 220, in _handle_execute_reply
    self.handle_exec_method(msg)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\shell.py", line 392, in handle_exec_method
    self._kernel_reply = ast.literal_eval(data['text/plain'])
  File "C:\ProgramData\Anaconda3\lib\ast.py", line 48, in literal_eval
    node_or_string = parse(node_or_string, mode='eval')
  File "C:\ProgramData\Anaconda3\lib\ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 3
    'name': <function pandas.core.groupby.GroupByPlot.__getattr__.<locals>.attr>,
            ^
SyntaxError: invalid syntax

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
n

**Comment by ccordoba12** [comment]
I think this was fixed in our 3.2.5 version, and our latest one is 3.2.6. Please update, try again and report back to see if your problem was fixed. Thanks!

**Comment by janvanlooy** [comment]
Indeed it was. Closing the issue.
